packages/infolists/resources/views/components/text-entry.blade.php

```diff
@@ -56,8 +58,17 @@
             >
                 <{{ $isListWithLineBreaks ? 'ul' : 'div' }}
                     @class([
+                        'flex flex-wrap items-center' => ! $isListWithLineBreaks,
                         'list-inside list-disc' => $isBulleted(),
-                        'flex flex-wrap items-center gap-1.5' => $isBadge,
+                        'gap-1.5' => $isBadge,
+                        match ($alignment) {
```

Comment:
Let's make `Alignment` a backed enum to simplify these match arms.

Reply:
@zepfietje  i worked on changing to backed enum, it seems to be breaking change
 
this is because even is backed, `Alignment::End != 'end'`
```php
// example with ActionSize::ExtraLarge dump
Filament\Support\Enums\ActionSize {
  +name: "ExtraLarge"
  +value: "xl"
}
```

as can seen in the codebase, there are multiple places you still using enum and string for the match
![image](https://github.com/filamentphp/filament/assets/67364036/f04a9147-9306-4f5d-8b94-4cb44729448e)


another way is to resolve the value back to enum 
```php
    public function getAlignment(): Alignment|string|null
    {
        $alignment = $this->evaluate($this->alignment);

        if (is_string($alignment)) {
            return Alignment::tryFrom($alignment);
        }

        return $alignment;
    }
```

Let me know if I'm understand you request wrong, or how you want to proceed with this

Reply:
i will leave this PR as draft waiting for your #9167

Reply:
Yeah sorry, was about to say I'm handling this in linked PR.
Requires quite a big refactor. 🙂